### DISCUSSION TITLE
使用InstructionWild数据中的seed_prompts_en.jsonl数据集对llama-7b模型进行预训练报错

### DISCUSSION THREAD

**Discussion by 13416157913** [hpcaitech/ColossalAI#4387] [Community | Q&A] [answered]
执行命令是：torchrun --standalone --nproc_per_node=1 train_sft.py --pretrain "/home/llm-deploy/ColossalAI/llama-7b" --model 'llama' --strategy colossalai_zero2 --log_interval 10 --save_path /home/llm-deploy/ColossalAI/output/llama-7B --dataset /home/llm-deploy/ColossalAI/InstructionWild/data/seed_prompts_en.jsonl --batch_size 1 --lr 2e-5 --max_epochs 1

报错信息如下：
<img width="960" alt="微信图片_20230807115148" src="https://github.com/hpcaitech/ColossalAI/assets/20581199/40e184f5-c633-40e9-8a97-fe31a74a4ed4">

**Comment by cwher** [comment]
It seems that you used the wrong dataset. 🤔

**Comment by cwher** [comment]
The dataset should follow the following format,
```bash
[
    {
        "instruction": "Give three tips for staying healthy.",
        "input": "",
        "output": "1.Eat a balanced diet and make sure to include plenty of fruits and vegetables. \n2. Exercise regularly to keep your body active and strong. \n3. Get enough sleep and maintain a consistent sleep schedule."
    },
...
```

**Comment by flybird11111** [comment, marked as answer]
> 执行命令是：torchrun --standalone --nproc_per_node=1 train_sft.py --pretrain "/home/llm-deploy/ColossalAI/llama-7b" --model 'llama' --strategy colossalai_zero2 --log_interval 10 --save_path /home/llm-deploy/ColossalAI/output/llama-7B --dataset /home/llm-deploy/ColossalAI/InstructionWild/data/seed_prompts_en.jsonl --batch_size 1 --lr 2e-5 --max_epochs 1
> 
> 报错信息如下： <img alt="微信图片_20230807115148" width="960" src="https://user-images.githubusercontent.com/20581199/258711195-40e184f5-c633-40e9-8a97-fe31a74a4ed4.png">

![FQe3qNQRAe](https://github.com/hpcaitech/ColossalAI/assets/37931082/7985ecc8-ba5a-4830-9cdd-533c634cd53d)
The training data is contained in these two files.
